compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs

```diff
@@ -407,16 +406,12 @@ Renders a FeatureVersionReq.
 "1.1 to 1.2, or 2.3 to 2.dev"
 -}
 renderFeatureVersionReq :: VersionReq -> String
-renderFeatureVersionReq (VersionReq req) = renderRanges (req V1) (req V2)
+renderFeatureVersionReq (VersionReq req) =
+  orStrs $ catMaybes
+    [ renderRange (Version v) (req v)
+    | v <- [minBound @MajorVersion .. maxBound]
```

Comment:
I'm surprise you need the type application here. (Or maybe you don't need it but added it for readability, I am more than fine with that.)

Reply:
ah yes, inference gets this one right but I prefer the explicit version for readability. 
At previous jobs we had `every :: forall a. (Enum a, Bounded a) => [a]; every = [minBound .. maxBound]` so at use sites it looked like `every @MajorVersion` :)